trade-finance-manager-ui/server/test-mocks/mock-banks.ts

```diff
@@ -14,31 +14,31 @@ const MONTHLY_REPORT_PERIOD_SCHEDULE = [
 ];
 
 export const MOCK_BANKS = {
-  BARCLAYS: {
+  TestBank1: {
```

Comment:
```suggestion
  TESTBANK1: {
```
to keep it consistent? views @abhi-markan 

However keep name as is

Reply:
Can we write these bank names in camel case `testBank1`?

Reply:
Done